sdk/eventhub/event-hubs/src/sender.ts

```diff
@@ -142,7 +142,7 @@ export class EventHubProducer {
   }
 
   /**
-   * Send one or more of events to the associated Event Hub.
+   * Send events to the associated Event Hub.
    *
    * @param eventData  An individual `EventData` object, or an array of `EventData` objects or an
```

Comment:
Nit since you updated this comment :smile:
```suggestion
   * @param eventData  An array of `EventData` objects or an
```